src/features/embalm/stepContent/components/ArchaeologistList.tsx

```diff
@@ -151,6 +167,18 @@ export function ArchaeologistList() {
                           </Text>
                         </Flex>
                       </Td>
+                      {
+                        includeDialButton ? (
+                          <Td>
+                            <Button
+                              disabled={isDialing || !!arch.connection}
+                              onClick={() => testDialArchaeologist(arch.fullPeerId!)}
+                            >
+                              { arch.connection ? 'Connected' : 'Dial' }
```

Comment:
note -- `!!arch.connection` does not ever get set to true (even when the connection property is updated in global state). This isn't an issue right now as the connection object isn't necessary for this functionality (and the connection setting does work during the create flow), but could potentially look into this item later.